src/storage/src/flush.rs

```diff
@@ -212,7 +212,7 @@ impl<S: LogStore> FlushJob<S> {
     async fn write_manifest_and_apply(&self, file_metas: &[FileMeta]) -> Result<()> {
         let edit = RegionEdit {
             region_version: self.shared.version_control.metadata().version(),
-            flushed_sequence: self.flush_sequence,
+            flushed_sequence: Some(self.flush_sequence),
```

Comment:
Do we need to ensure that files on the same level use the same time window size? Then we might need to persist the time window size into the manifest.

Reply:
I create a future work item in https://github.com/GreptimeTeam/greptimedb/issues/930